.travis.yml

```
@@ -69,3 +69,5 @@ jobs:
             php: 7.2
         -   <<: *TEST_MYSQL
             php: 7.3
+        -   <<: *TEST_MYSQL
+            php: 7.4
```

```suggestion
            php: 7.4snapshot
```
Ich meine gelesen zu haben dass die 7.4 builds relativ alt sind, wenn man nicht mit snapshot arbeitet

---

`7.4` scheint es noch gar nicht zu geben